sys/linux/socket_netlink_route.txt

```diff
@@ -850,6 +878,58 @@ br_mdb_entry_sub_u [
 	in6_addr	ipv6_addr
 ]
 
+rtm_nh_policy_get_id [
+	NHA_ID	nlattr[NHA_ID, int32[0:2]]
+] [varlen]
+
+rtm_nh_policy_get_dump [
+	NHA_OIF		nlattr[NHA_OIF, ifindex]
+	NHA_GROUPS	nlattr[NHA_GROUPS, void]
+	NHA_MASTER	nlattr[NHA_MASTER, int32[0:2]]
+	NHA_FDB		nlattr[NHA_FDB, void]
+] [varlen]
+
+rtm_nh_policy_new [
+	NHA_ID		nlattr[NHA_ID, int32[0:2]]
+	NHA_GROUP	nlattr[NHA_GROUP, array[nexthop_grp]]
+	NHA_GROUP_TYPE	nlattr[NHA_GROUP_TYPE, flags[nha_group_types, int16]]
+	NHA_OIF		nlattr[NHA_OIF, ifindex]
+	NHA_GATEWAY	nlattr[NHA_GATEWAY, nha_gateway]
+	NHA_ENCAP_TYPE	nlattr[NHA_ENCAP_TYPE, flags[lwtunnel_encap_types, int16]]
+	NHA_ENCAP	nlnest[NHA_ENCAP, lwtunnel_policy]
+] [varlen]
+
+rtm_nh_policy_new_fdb [
+	NHA_ID		nlattr[NHA_ID, int32[0:2]]
+	NHA_GROUP	nlattr[NHA_GROUP, array[nexthop_grp]]
+	NHA_GROUP_TYPE	nlattr[NHA_GROUP_TYPE, flags[nha_group_types, int16]]
+	NHA_GATEWAY	nlattr[NHA_GATEWAY, nha_gateway]
+	NHA_FDB		nlattr[NHA_FDB, void]
+] [varlen]
+
+rtm_nh_policy_new_blackhole [
+	NHA_ID		nlattr[NHA_ID, int32[0:2]]
+	NHA_GROUP	nlattr[NHA_GROUP, array[nexthop_grp]]
+	NHA_GROUP_TYPE	nlattr[NHA_GROUP_TYPE, flags[nha_group_types, int16]]
+	NHA_BLACKHOLE	nlattr[NHA_BLACKHOLE, void]
+] [varlen]
```

Comment:
Like above, is there any gain in seperating this?

As you mentioned in your comments, we can merge these and have a single `new` policy same as before.